_drafts/git-bisect.md

```diff
@@ -0,0 +1,600 @@
+---
+title: "Hunting Bugs with Git Bisect"
+description: "An introduction to bisecting a set of changes as a bug-hunting tool, followed by some deeper techniques to speed up the process of tracking down when behavior is introduced"
+author: shiva
+image: /img/git-bisect/git-logo.png
+---
+<style>
```

Comment:
Woops I need to nuke this

Reply:
Went ahead and removed it myself in 81febd407.